src/test/typescript/mockserver/deposit.ts

```diff
@@ -50,7 +50,7 @@ export const depositData3: Deposit = {
 export const depositData4: Deposit = {
     title: "Old Dataset",
     state: "ARCHIVED",
-    stateDescription: "easy-dataset:id",
+    stateDescription: "The dataset is published at <a href='https://deasy.dans.knaw.nl/ui/datasets/id/easy-dataset:11' target='_blank'>https://deasy.dans.knaw.nl/ui/datasets/id/easy-dataset:11</a>",
```

Comment:
If this is also used in the deposit-overview page, the deposit-api would no longer need to construct the link and the application properties of the deposit-api can forget its knowledge about the webui.

Reply:
this is just a test object... The API should serve a similar kind of object.